backend/app/api/endpoints/export.py

```
@@ -94,6 +99,9 @@ async def export_book_pdf(
             }
         )
 
+    except ExportUnavailableError as e:
+        logger.warning("PDF export unavailable: %s", e)
+        raise HTTPException(status_code=503, detail=str(e))
```

Ruff (0.15.18)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)